diff --git a/node/test/bundle.test.mjs b/node/test/bundle.test.mjs
index 4279e51..6ce1956 100644
--- a/node/test/bundle.test.mjs
+++ b/node/test/bundle.test.mjs
@@ -346,7 +346,7 @@ test('read return non-string', async () => {
   }
 
   if (!error) throw new Error(`\`testReadReturnNonString()\` failed. Expected \`bundleAsync()\` to throw, but it did not.`);
-  assert.equal(error.message, 'expect String, got: Number');
+  assert.equal(error.message, 'Failed to convert JavaScript value `Number 1234 ` into rust type `String`');
 });
 
 test('resolve return non-string', async () => {
