diff --git a/selectors/parser.rs b/selectors/parser.rs
index d79365b..19563d5 100644
--- a/selectors/parser.rs
+++ b/selectors/parser.rs
@@ -880,12 +880,12 @@ impl<'i, Impl: SelectorImpl<'i>> Selector<'i, Impl> {
   /// Returns an iterator over the entire sequence of simple selectors and
   /// combinators, in matching order (from right to left).
   #[inline]
-  pub fn iter_raw_match_order(&self) -> slice::Iter<Component<'i, Impl>> {
+  pub fn iter_raw_match_order(&self) -> slice::Iter<'_, Component<'i, Impl>> {
     self.1.iter()
   }
 
   #[inline]
-  pub fn iter_mut_raw_match_order(&mut self) -> slice::IterMut<Component<'i, Impl>> {
+  pub fn iter_mut_raw_match_order(&mut self) -> slice::IterMut<'_, Component<'i, Impl>> {
     self.1.iter_mut()
   }
 
@@ -903,7 +903,7 @@ impl<'i, Impl: SelectorImpl<'i>> Selector<'i, Impl> {
   /// combinators, in parse order (from left to right), starting from
   /// `offset`.
   #[inline]
-  pub fn iter_raw_parse_order_from(&self, offset: usize) -> Rev<slice::Iter<Component<'i, Impl>>> {
+  pub fn iter_raw_parse_order_from(&self, offset: usize) -> Rev<slice::Iter<'_, Component<'i, Impl>>> {
     self.1[..self.len() - offset].iter().rev()
   }
 
diff --git a/src/compat.rs b/src/compat.rs
index d387fba..3b29a9f 100644
--- a/src/compat.rs
+++ b/src/compat.rs
@@ -5742,7 +5742,6 @@ impl Feature {
       if self.is_compatible(browsers) {
         return true;
       }
-      browsers.samsung = None;
     }
 
     false
diff --git a/src/properties/custom.rs b/src/properties/custom.rs
index 83a2927..cdc20ce 100644
--- a/src/properties/custom.rs
+++ b/src/properties/custom.rs
@@ -1081,7 +1081,7 @@ impl<'a> std::hash::Hash for Token<'a> {
 /// Converts a floating point value into its mantissa, exponent,
 /// and sign components so that it can be hashed.
 fn integer_decode(v: f32) -> (u32, i16, i8) {
-  let bits: u32 = unsafe { std::mem::transmute(v) };
+  let bits = v.to_bits();
   let sign: i8 = if bits >> 31 == 0 { 1 } else { -1 };
   let mut exponent: i16 = ((bits >> 23) & 0xff) as i16;
   let mantissa = if exponent == 0 {
